pulsar-broker/src/test/java/org/apache/pulsar/schema/compatibility/SchemaTypeCompatibilityCheckTest.java

```diff
@@ -101,9 +117,44 @@ public void testSchemaCompatibilityStrategyInBrokerLevel() throws PulsarClientEx
         assertTrue(t.getMessage().contains("org.apache.avro.SchemaValidationException: Unable to read schema"));
     }
 
+    @Test
+    public void testSchemaCompatibilityStrategyInTopicLevel()
```

Comment:
Please add a unit test to test the priority between different levels of strategies